contracts/solana/stork/programs/stork/src/lib.rs

```diff
@@ -0,0 +1,244 @@
+use anchor_lang::prelude::*;
+use anchor_lang::solana_program::system_instruction;
+
+mod verify;
+use verify::{verify_stork_evm_signature, EvmPubkey};
+
+declare_id!("F8C2ffLfHGQTQxnix4LaroXuHh2Cdv8wJw7R9UDEfmX1");
+
+pub const STORK_CONFIG_SEED: &[u8] = b"stork_config";
+pub const STORK_FEED_SEED: &[u8] = b"stork_feed";
+pub const STORK_TREASURY_SEED: &[u8] = b"stork_treasury";
+
+#[program]
+pub mod stork {
+    use super::*;
+
+    pub fn initialize(
+        ctx: Context<Initialize>,
+        stork_sol_public_key: Pubkey,
+        stork_evm_public_key: EvmPubkey,
+        single_update_fee_in_lamports: u64,
+    ) -> Result<()> {
+        let config = &mut ctx.accounts.config;
+        config.stork_sol_public_key = stork_sol_public_key;
+        config.stork_evm_public_key = stork_evm_public_key;
+        config.single_update_fee_in_lamports = single_update_fee_in_lamports;
+        config.owner = ctx.accounts.owner.key();
+        Ok(())
+    }
+
+    pub fn update_temporal_numeric_value_evm(
+        ctx: Context<UpdateTemporalNumericValue>,
+        update_data: TemporalNumericValueEvmInput,
+    ) -> Result<()> {
+        let config = &ctx.accounts.config;
+        let treasury = &ctx.accounts.treasury;
+        let feed = &mut ctx.accounts.feed;
+        let payer = &ctx.accounts.payer;
+
+        if feed.id == [0; 32] {
+            feed.id = update_data.id;
+        }
+
+        if !verify_stork_evm_signature(
+            &config.stork_evm_public_key,
+            update_data.id,
+            update_data.temporal_numeric_value.timestamp_ns,
+            update_data.temporal_numeric_value.quantized_value,
+            update_data.publisher_merkle_root,
+            update_data.value_compute_alg_hash,
+            update_data.r,
+            update_data.s,
+            update_data.v,
+        ) {
+            return Err(StorkError::InvalidSignature.into());
+        }
+
+        require_gte!(
+            update_data.temporal_numeric_value.timestamp_ns,
+            feed.latest_value.timestamp_ns,
+            StorkError::NoFreshUpdate
+        );
+
+        let amount_to_pay = config.single_update_fee_in_lamports;
+        if payer.lamports()
+            < Rent::get()?
+                .minimum_balance(payer.data_len())
+                .saturating_add(amount_to_pay)
+        {
+            return err!(StorkError::InsufficientFunds);
+        };
+        let transfer_instruction =
+            system_instruction::transfer(payer.key, treasury.key, amount_to_pay);
+        anchor_lang::solana_program::program::invoke(
+            &transfer_instruction,
+            &[payer.to_account_info(), treasury.to_account_info()],
```

Comment:
One line here too, or have the last argument of invoke() on a different line. Also since you're already importing anchor_lang::solana_program::system_instruction, it would probably make sense to just import program too in the same line (use  anchor_lang::solana_program::{system_instruction, program}) and cut out needing to use the path on program.